packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.ts

```diff
@@ -61,7 +61,9 @@ export default ({ api, coreSagas, networks }) => {
   const LOGIN_FORM = 'login'
 
   const authNabu = function* () {
-    yield put(actions.components.identityVerification.fetchSupportedCountries())
+    yield put(
+      actions.components.identityVerification.fetchSupportedCountries({ scope: CountryScope.KYC })
+    )
```

Comment:
why do we need to fetch country list here, is there a better spot?  this function is for authentication to nabu, seems like breaking single responsibility principles

Reply:
I have to check with @TheLeoB she added this in the first place, here I'm just changing call signature since we now pass scope parameter.